tox.ini

```diff
@@ -4,28 +4,16 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py37, py38, py39, py310, pypy3.7, flake8
+envlist = py37, py38, py39, py310, pypy3.7, lint
```

Comment:
```suggestion
envlist = lint, py37, py38, py39, py310, pypy3.7
```

Reply:
No, you add `depends = lint` to the testenv to declare the dependency. Then you add `depends =` to clear the dependency in the lint job itself to eliminate the cyclic reference.

Reply:
Ahh, okay. Will try that.